database/src/main/kotlin/no/nav/su/se/bakover/database/DatabaseEx.kt

```diff
@@ -13,9 +13,23 @@ private fun sjekkUgyldigParameternavn(params: Map<String, Any?>) {
     require(params.keys.none { it.contains(Regex("[æÆøØåÅ]")) }) { "Parameter-mapping forstår ikke særnorske tegn" }
 }
 
+private fun sjekkAtOppdaterInneholderWhere(sql: String) {
+    require(sql.lowercase().contains("where")) { "Ikke lov med update uten where" }
+    require(!sql.lowercase().contains("insert")) { "Ikke lov med insert i en update" }
+}
+
 internal fun String.oppdatering(
     params: Map<String, Any?>,
     session: Session
+) {
+    sjekkUgyldigParameternavn(params)
+    sjekkAtOppdaterInneholderWhere(this)
+    session.run(queryOf(statement = this, paramMap = params).asUpdate)
+}
+
+internal fun String.insert(
```

Comment:
"".insert(...) og "".oppdatering(...) klinger ikke så godt sammen. Kan godt ha insert og update for min del (så det blir en teknisk referanse til sql. Hvis vi vil ha enda-sterkere, "sqlInsert" og "sqlUpdate"